inventory_valuate_date_hls/wizards/stock_quantity_history.py

```diff
@@ -0,0 +1,13 @@
+from odoo import fields, models
+
+
+class StockQuantityHistory(models.TransientModel):
+    _inherit = "stock.quantity.history"
+
+    date_range = fields.Many2one("date.range", string="Date Range")
+    inventory_valuate_date = fields.Date(
+        string="Inventory Valuate Date",
+        related="date_range.inventory_valuate_date",
+        readonly=True,
+        store=True,
+    )
```

Comment:
@yostashiro @tack-yabeQuartile @kakurai8 
Sorry, I added this code but couldn't add field.
Please give me some advise.